classes/locations.php

```diff
@@ -51,6 +51,12 @@ public function register_locations() {
 			);
 		}
 
+		// Rewrite rules if we've changed the location permalink slug.
+		if ( get_option( 'sm-rewrite-rules' ) ) {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected 12 spaces, found 8
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->